### PR TITLE
Fixes #28231 - Package build time to human readable format

### PR DESCRIPTION
**Pull request by dhananjay316**

### CODE REVIEW

**Comment by sjha4** [comment]
[test katello]

File: app/views/katello/api/v2/packages/backend.json.rabl [hunk 1]
```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
+   Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
```
Comment (snagoor) [inline app/views/katello/api/v2/packages/backend.json.rabl]: @dhananjay316 an extra space added in the front, it should only have 2 spaces. Rubocop doesn't like it 
~~~
[2019-11-18T21:41:37.191Z] Offenses:
[2019-11-18T21:41:37.191Z] 
[2019-11-18T21:41:37.191Z] /home/jenkins/workspace/katello-pr-test/app/views/katello/api/v2/packages/backend.json.rabl:10:1: C: Layout/IndentationWidth: Use 2 (not 3) spaces for indentation.
[2019-11-18T21:41:37.191Z]    Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
[2019-11-18T21:41:37.191Z] ^^^
[2019-11-18T21:41:37.191Z] /home/jenkins/workspace/katello-pr-test/app/views/katello/api/v2/packages/backend.json.rabl:12:1: C: Layout/TrailingBlankLines: 1 trailing blank lines detected.
~~~

File: app/views/katello/api/v2/packages/backend.json.rabl [hunk 2]
```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
+  Time.at(package.build_time).strftime("%B %e, %Y at %I:%M %p") if package.build_time
+end
+
```
Comment (sjha4) [inline app/views/katello/api/v2/packages/backend.json.rabl]: We can avoid formatting here and use the date components on UI for consistency..Somnething like:
```
<date date = "package.human_readable_time" />
```

Comment (jturel) [reply app/views/katello/api/v2/packages/backend.json.rabl]: I think we could probably reference `package.build_time` directly in the UI through the <date> tag

Comment (sjha4) [reply app/views/katello/api/v2/packages/backend.json.rabl]: That might not work as the date component does not expect epoch time but I am not sure.

Comment (jturel) [reply app/views/katello/api/v2/packages/backend.json.rabl]: Ah ok!

Side note: I think human_readable_time should be moved into the ruby code (on the package model) and not in the RABL API template

File: app/views/katello/api/v2/packages/backend.json.rabl [hunk 3]
```diff
@@ -5,3 +5,8 @@ attributes :children, :checksumtype, :size, :url, :build_time, :summary, :group,
 node :human_readable_size do |package|
   number_to_human_size(package.size) if package.size
 end
+
+node :human_readable_time do |package|
```
Comment (jturel) [inline app/views/katello/api/v2/packages/backend.json.rabl]: Let's change this to UTC time, and the view can turn that into human readable time, like this:

here:
```
node :build_time_utc do |package|                                                                                                                                                             
  Time.at(package.build_time).to_datetime if package.build_time                                                                                                                               
end
```

in the view:

```
<dd><long-date-time date="package.build_time_utc" /></dd>
```

Comment (dhananjay316) [reply app/views/katello/api/v2/packages/backend.json.rabl]: @jturel Thank you for the suggestion. I have updated the mentioned changes.

**Comment by jturel** [comment]
[test katello]

**Comment by jturel** [comment]
Looks great, thank you @dhananjay316 !

**Pull request merged**